go_api/interfaces/admin_controller.go

```diff
@@ -0,0 +1,91 @@
+// AdminのControllerを定義する
+package controller
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/NUTFes/lottery/go_api/domain"
+	"github.com/NUTFes/lottery/go_api/usecase"
+	"github.com/labstack/echo/v4"
+)
+
+type adminController struct {
+	adminUsecase usecase.AdminUsecase
+}
+
+type AdminController interface {
+	IndexAdmin(c echo.Context) error
+	ShowAdmin(c echo.Context) error
+	CreateAdmin(c echo.Context) error
+	UpdateAdmin(c echo.Context) error
+	DeleteAdmin(c echo.Context) error
+}
+
+func NewAdminController(uu usecase.AdminUsecase) AdminController {
+	return &adminController{adminUsecase: uu}
+}
+
+// 全アドミンユーザーの取得
+func (u *adminController) IndexAdmin(c echo.Context) error {
+	admins, err := u.adminUsecase.FindAllAdmin()
+	if err != nil {
+		return err
+	}
+	return c.JSON(http.StatusOK, admins)
+}
```

Comment:
```suggestion
// 全アドミンユーザーの取得
func (a *adminController) IndexAdmin(c echo.Context) error {
	admins, err := a.adminUsecase.FindAllAdmin()
	if err != nil {
		return err
	}
	return c.JSON(http.StatusOK, admins)
}
```
このファイルのレシーバーも修正お願いします